server/routerlicious/packages/lambdas-driver/src/kafka-service/partitionManager.ts

```diff
@@ -58,13 +58,11 @@ export class PartitionManager extends EventEmitter {
 			});
 
 			this.consumer.on("checkpoint_success", (partitionId, queuedMessage) => {
-				if (this.sampleMessages(100)) {
-					Lumberjack.info(`Kafka checkpoint successful`, {
```

Comment:
Kara, as you worried, we indeed generate a lot of logs for Kafka commit/checkpoint. RawOpLogger alone had more than 100K in the past day in some clusters. If the sampling works as expected, it means we have committed  more than10M times only from RawOpLogger! To avoid making matters worse, let's hold off on logging every commit until we address the excessive commits.